bin/make_idl_files.py

```diff
@@ -41,6 +41,14 @@
     components = lsst.ts.xml.subsystems
 else:
     components = args.components
+    invalid_components = [
+        component for component in components if component not in lsst.ts.xml.subsystems
+    ]
+    if len(invalid_components):
```

Comment:
I found this a bit hard to read. I think it would be more idomatic to skip the len:
```
if invalid_components:
   ...
```
Another (more verbose) option is to explicitly compare the length against 0:
```
if len(invalid_components)> 0:
    ...
```

Reply:
great catch! I forgot the `> 0`